codespell_lib/data/dictionary.txt

```diff
@@ -33042,6 +33125,7 @@ unabel->unable
 unablet->unable
 unacceptible->unacceptable
 unaccesible->unaccessible
+unaccessable->unaccessible
```

Comment:
@luzpaz  For both misspelling `unaccesible` and `unaccessable`, can you suggest `inaccessible`?
Major online dictionaries, hunspell, aspell, Chrome browser, etc considered `unaccessable` as a misspelling.

https://grammarhow.com/unaccessible-vs-inaccessible/
"`Unaccessible` is considered as improper and `inaccessible` should always be used in a formal or professional setting."

https://www.lexico.com/search?filter=en_dictionary&query=unaccessible
https://www.ldoceonline.com/spellcheck/english/?q=unaccessible
https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=unaccessible
https://dictionary.cambridge.org/spellcheck/english/?q=unaccessible

```text
unaccesible->inaccessible
unaccessable->inaccessible
```

Reply:
[432309b](https://github.com/codespell-project/codespell/pull/2412/commits/432309b91a91b9087bf4d89ba0701912fac87e40)